joomla-projects/privacy-framework
# Fix wrong items link in action logs

Pull Request for Issue #159.

### Summary of Changes
This PR fixes wrong links to item edit page of several content types in action logs (Menu item, access level, banner client). Basically, instead of using type_title (menu_item, access_level, banner_client), we use type_alias (com_menus.item, com_users.level, com_banners.client) to detect content type to generate link to item edit page of each content type.

### Testing Instructions
1. Add new access level
2. Access to action logs component, note that the link to access level is causing error
3. Apply patch, add new access level, confirm that the link to that access level in action logs is now correct
4. Add new article, then check and make sure link to that article in action log is correct, too.
